chart/compass/charts/external-services-mock/templates/tenant-dependencies-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if eq .Values.global.externalServicesMock.tenantDependenciesConfig.manage true }}
```

Comment:
Can you please wrap the condition in if in brackets, like this:
{{- if (eq .Values.global.externalServicesMock.tenantDependenciesConfig.manage true) }}